pallet/src/lib.rs

```diff
@@ -570,7 +598,7 @@ pub mod pallet {
             let Ok(hashes_ready_for_cleanup) = ChoreOnIdleStorage::<T>::try_get(block) else {
                 return T::DbWeight::get().reads(1);
             };
-            // We don't need this entry in storage anymore, remove it!
+            // We don't need this entry in storage anymore, remove it.
```

Comment:
:laughing:

resolve when read :+1:

but: :laughing:

Reply:
I don't like to yell to my reviewers :D